dcf/admin.py

```diff
@@ -30,7 +30,7 @@ class SectionAdmin(admin.ModelAdmin):
 
 
 class ComplaintAdmin(admin.ModelAdmin):
```

Comment:
Needs to be Py2 compatible, please use `@python_2_unicode_compatible` class decorator.